framework/console/CConsoleCommand.php

```diff
@@ -229,6 +229,9 @@ protected function resolveRequest($args)
 		$params=array();	// unnamed parameters
 		foreach($args as $arg)
 		{
+            if (is_null($arg))
+                continue;
+
```

Comment:
Yii uses tabs instead of spaces.

Reply:
Fixed.

Reply:
2. To reproduce this issue, simply create your own command that inherits from ConsoleCommand and uses resolveRequest() for parsing arguments, like:
`[$action, $options, $args] = $this->resolveRequest($this->arguments);`
Then, when you write unit test for tis command, you may call it directly with wrong attributes (for negative testing), something like that:
```
$cmd = new OptimizeCommand(null, null);
$cmd->run(['css', '-s', null], false);
```
If one of arguments will be a null, you get an error: 
`PHP Deprecated: preg_match(): Passing null to parameter https://github.com/yiisoft/yii/issues/2 ($subject)`

Reply:
I don't think there normally is a reason you should be able to pass null to a CLI command argument, but the change is valid and keeps backward compatibility.